index.html

```diff
@@ -24,7 +25,7 @@ <h1>Lambda School Newsfeed</h1>
     </div>
     <div class="articles">
       <div class="article">
-        <h2>Lambda School Students: "We're the best!"</h2>
+        <h2>IDK WHAT I'M DOING</h2>
```

Comment:
I like that you have fun w/ the project and make it your own.

Reply:
lol thanks man